src/main/java/mate/academy/spring/validation/Email.java

```diff
@@ -0,0 +1,17 @@
+package mate.academy.spring.validation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import javax.validation.Constraint;
+import javax.validation.Payload;
+
+@Constraint(validatedBy = EmailValidator.class)
+@Target({ElementType.PARAMETER, ElementType.FIELD})
```

Comment:
do we need a parameter target?

Reply:
my bad

Reply:
actually, you can put it over the parameter in the `find user by email` method but I wouldn't say that it is necessary